commands/gban.js

```diff
@@ -23,6 +23,12 @@ exports.run = async (client, message, args) => {
 
     const potentialGuilds = client.guilds.filter(g=>g.me.hasPermission('BAN_MEMBERS'))
 
+    let completemessage;
```

Comment:
Unnecessary

Reply:
No, not really.
It's to make the value null if the message can't send.